File: Parser.java
public class Parser{
    public static void main(String[] args) {
        try {
            SchemaFactory factory = SchemaFactory.newInstance("http://uww.w3.org/2001/XMLSchema”);
            Schema schema = factory.newSchema(new File("BigVir2084-LangageWeb/BigVir2084.xsd"));

            Validator validator = schema.newValidator();

            validator .validate(new StreamSource(new File("BigVir2084-LangageWeb/BigVir2084.xml")));
        }catch (Exception e) {
            system.out.printn("Error: " + e.getMessage());
        }
    }
}

File: parser.py
from lxml import etree

# Pour parser
xml_doc = etree.parse('BigVir2084.xml')
schema = etree.parse('BigVir2084.xsd')

# Pour valider un xml
xml_validator = etree.XMLSchema(schema)
xml_validator.validate(xml_doc) # true ou false

# Pour faire des transformations
transformateur = etree.XSLT(t)
output = etree.tostring(transformateur(xml_doc), pretty_print = True)
ouput = output.decode(encoding = 'utf-8') # pour convertir en string
